### PR TITLE
templates: opera news

### PR DESCRIPTION
**Pull request by atrisovic**
* Closes: #1248 
  * Closes: #1954
  * Closes: #1953

Signed-off-by: Ana Trisovic <ana.trisovic@gmail.com>

### CODE REVIEW

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-If you are interested in conducting a research-oriented analysis using CMS Open Data, you should find hints, tips and guidance on this page. Please note that possible solutions to frequently encountered issues can be found on [our page of known errors]( /articles/cms-guide-to-troubleshooting).
+If you are interested in step-by-step instructions to start working with CMS Open Data, please consult these pages:
+
+* [Install Virtual Machine](/articles/cms-2011-virtual-machines-how-to-install)
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: /docs

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+* [Get started with CMS Open Data](/articles/getting-started-with-cms-2011-open-data)
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: same

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 3]
```diff
@@ -51,7 +56,7 @@ If you are interested in conducting a research-oriented analysis using CMS Open
 > "I want to produce some example physics distributions (e.g. inclusive di-muon spectrum analysis example directly from AOD or two-/four-lepton analysis example with intermediate ntuples)."
 
 - Install the CMS software as in the previous item (for 2010 or 2011).
-- Follow options A (inclusive di-muon spectrum analysis example directly from AOD) or B (two-/four-lepton analysis example with intermediate ntuples) in "Getting started with CMS 2010 open data"(/articles/getting-started-with-cms-2010-open-data) or ["Getting started with CMS 2011 open data"](/articles/getting-started-with-cms-2011-data). 
+- Follow options A (inclusive di-muon spectrum analysis example directly from AOD) or B (two-/four-lepton analysis example with intermediate ntuples) in ["Getting started with CMS 2010 open data"](/articles/getting-started-with-cms-2010-open-data) or ["Getting started with CMS 2011 open data"](/articles/getting-started-with-cms-2011-open-data).
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: same

Comment (atrisovic) [reply cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: fine! 👍

**Comment by atrisovic** [comment]
Ok done ✅

**Pull request merged**